lib/src/upgrader.dart

```diff
@@ -44,6 +45,11 @@ class Upgrader {
   /// When an appcast is configured for iOS, the iTunes lookup is not used.
   AppcastConfiguration? appcastConfig;
 
+  /// An optional value that can override the default packageName when
+  /// attempting to reach the Google Play Store. This is useful if your app has
+  /// a different package name in the Play Store.
```

Comment:
Can you provide an example when the package name does not match the Play Store?

Reply:
The android ID is asked as an optional value and will be used Incase the package name does not exist.

final id = androidId ??_packageInfo!.packageName;

Reply:
When packageName does exist, should packageName be used instead of `androidId`? Can you show me an example of when packageName does not exist for an Android app?

Reply:
I don't think this is possible. Can you show me an example of when packageName does not exist for an Android app?

Reply:
@Williano I don't think this is possible. Can you show me an example of when packageName does not exist for an Android app?